### PR TITLE
Generate supported-backends markdown table

### PR DESCRIPTION
**Pull request by kaizhong2021**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/05-supported-backends/generate_backends_md_table.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+This tutorial describes how to generate supported-backends markdown table.
+
+## 1.python Environment
+
+```
+pip install easydict
+```
+
+## 2.Usage
```
Comment (RunningLeon) [inline docs/en/05-supported-backends/generate_backends_md_table.md:11]: usage and description should change the format  .could refer to https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/useful_tools.md#torch2onnx

File: docs/en/05-supported-backends/generate_backends_md_table.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# generate_md_table
+
+This tool can be used to generate supported-backends markdown table.
+
+## python Environment
+
+```
+pip install easydict
```
Comment (RunningLeon) [inline docs/en/05-supported-backends/generate_backends_md_table.md:8]: could move this to `requirements/test.txt`

File: tests/regression/generate_md_table.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+
+def main():
+    args = parse_args()
+    assert osp.exists(args.yml_file), f'File not exists: {args.yml_file}'
+    output_dir, _ = osp.split(args.output)
+    if output_dir:
+        os.makedirs(output_dir, exist_ok=True)
+    backends = [
+        'onnxruntime', 'tensorrt', 'torchscript', 'pplnn', 'openvino', 'ncnn'
```
Comment (lvhan028) [inline tests/regression/generate_md_table.py:28]: There are more inference backends supported besides those 5, like huawei's CANN, apple's CoreML, Rockchip's RKNN

Comment (RunningLeon) [reply tests/regression/generate_md_table.py]: @lvhan028 Personally, too much unsupported models for this kind of backends, which makes the table too large.
Besides, if we really want to include these kind of backends, it's better to do this by those who supported the backends and let them update the regression yam config files.

File: tests/regression/mmcls.yml [hunk 4]
```diff
@@ -1,4 +1,5 @@
 globals:
+  repo_url: https://github.com/open-mmlab/mmclassification/tree/master
```
Comment (RunningLeon) [inline tests/regression/mmcls.yml:2]: may sync with latest master and change `mmaction2.yml` as well.

File: docs/en/05-supported-backends/generate_backends_md_table.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# generate_md_table
+
+This tool can be used to generate supported-backends markdown table.
+
+## Usage
+
+```
```
Comment (grimoire) [inline docs/en/05-supported-backends/generate_backends_md_table.md:7]: special the type of the codeblock.

File: tests/regression/generate_md_table.py [hunk 6]
```diff
@@ -0,0 +1,76 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```
Comment (grimoire) [inline tests/regression/generate_md_table.py:1]: Should we place this in tools instead of tests?

File: tests/regression/generate_md_table.py [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    return args
+
+
+def main():
+    args = parse_args()
+    assert osp.exists(args.yml_file), f'File not exists: {args.yml_file}'
+    output_dir, _ = osp.split(args.output)
+    if output_dir:
+        os.makedirs(output_dir, exist_ok=True)
+    backends = [
```
Comment (grimoire) [inline tests/regression/generate_md_table.py:27]: Could be an argument.

**Comment by kaizhong2021** [comment]
> @kaizhong2021 Hi, could you update this PR according to the comments?

Thanks teacher ,I will fix it later.

File: docs/en/05-supported-backends/generate_backends_md_table.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+- `yml_file:` input yml config path
+- `output:`  output markdown file path
+- `backends:` output backends list. If not specified, it will be set 'onnxruntime' 'tensorrt' 'torchscript' 'pplnn' 'openvino' 'ncnn'.
+
+## Example:
+
+Generate backends markdown table from mmocr.yml
+
+```
```
Comment (grimoire) [inline docs/en/05-supported-backends/generate_backends_md_table.md:24]: Add type for the codeblock.
And generate_md_table.py has been moved to tools.

**Review by RunningLeon — APPROVED** [review]
LGTM

File: docs/en/05-supported-backends/generate_backends_md_table.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+# generate_md_table
```
Comment (grimoire) [inline docs/en/05-supported-backends/generate_backends_md_table.md:1]: I don't think this doc should be placed in supported-backends. Any advice @RunningLeon

Comment (RunningLeon) [reply docs/en/05-supported-backends/generate_backends_md_table.md:1]: Yes. Could be useful tools: https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/useful_tools.md

**Review by grimoire — APPROVED** [review]
LGTM

**Pull request merged**